### DISCUSSION TITLE
A Guide to Modernizing the WotLK Client

### DISCUSSION THREAD

**Discussion by Marotheit** [chromiecraft/chromiecraft#2679] [Guides]
# The guide has been moved [here](https://github.com/Marotheit/A-Guide-to-Modernizing-the-WotLK-Client) as to unassociate the server with any issues that may arise by using the patches listed in the guide.

# Game on. Long live ChromieCraft.

**Comment by 55Honey** [comment]
While we didn't approve the content, we certainly approve the effort put into it. Cheers! Want to put it on our discord, too?

We recommend watching out for model changes though. There are W.o.D. models available for download which we link to on our website. We can not guarantee that any other client changes are not sanctioned automatically.

**Reply by Marotheit** [reply]
I threw a link to it in the `#💽 | addons` channel, feel free to pin it!

That's very fair, regarding model changing. I've been using all of these patches for the past two weeks (with the exception of the Blood Mod and the ones I've personally modified) and haven't had any issues yet, but it was a concern, so I'll second that information.

**Comment by Marotheit** [comment]
Changelog
---
## As of January 7th, 2023, the guide has been moved [here](https://github.com/Marotheit/A-Guide-to-Modernizing-the-WotLK-Client) as to unassociate the server with any issues that may arise by using the patches listed in the guide. In doing so, I performed a once-over and fixed some wording and grammatical errors. Game on. Long live ChromieCraft.

(12.31.2021) Removed links to LoginTweaks and Updated Spell Visuals in an effort to curb triggering the server anti-cheat. Adjusted the values changed in the Ultra+ Graphics macro. This time, you simply have to update the text of the first macro and re-run the macro.

(12.28.2021) Fixed a conflict between the Ultra+ Graphics macro and patches that change textures. You should reset the changes applied before (change a graphic option in the 'Effects' section of your video settings), re-copy the macros, and finally re-run the macros to apply all changes. Added information regarding LoginTweaks and [ElvUI-Glues-WotLK](https://github.com/Marotheit/ElvUI-Glues-WotLK).

(12.27.2021) Fixed an error in the 'Windowed Fullscreen' section. It should now function properly.

**Reply by doctorwizz** [reply]
Question: If say I do the macros like /console environmentDetail 150, is that stored in the config.wtf?
Great info Thx!

**Reply by Marotheit** [reply]
> 

Yes! It is stored alongside your regular options (resolution, refresh rate, sound levels, etc.). You could manually go into that file and remove or edit values if you'd like.

**Reply by doctorwizz** [reply]
Ahh great thx!

**Comment by 55Honey** [comment]
Sadly, some of these changes have led to automated detection as cheats. We strongly discourage using any client modifications not listed on our homepage.

**Reply by doctorwizz** [reply]
Which ones?

**Reply by 55Honey** [reply]
Since they were used combined, it's not possible to point a finger at one of them.

**Reply by doctorwizz** [reply]
I only use the login screen tweak that saves your password with ChromieCraft. Could that be detected as a cheat?
Also,I only use the patches listed on the CC website.

**Reply by 55Honey** [reply]
I'm sorry, but we do not support client modifications of any kind and can not guarantee any of the work.

**Reply by Marotheit** [reply]
Big oof. If given the tools, I would love to help figure out what is acceptable and what's not allowed. If I setup AzerothCore on my PC, would the applicable anti-cheat be triggered there so I could see what was causing the issue? That being said, I imagine it's the updated spell visuals, which I'll be removing from my client immediately. I just got my Reins of the Swift Spectral Tiger and would love to keep it.

Edit: I've removed Updated Spell Visuals and LoginTweaks from the guide in an effort to comply with the anti-cheat. If anything triggered them, it would most likely be one of those two. Almost everything else is texture changes.

**Reply by 55Honey** [reply]
>  If I setup AzerothCore on my PC, would the applicable anti-cheat be triggered there so I could see what was causing the issue?

Of course, that would work.
Texture changes aren't supposed to trigger anything.

**Reply by Saikur0n** [reply]
Unfortunately that ChromieCraft will not support patches with spell visual changes. In CVPatch collection there is a such good patch to modify visuals of spells and abilities of classes that gives fresh and modern looking to old game. May be it is possible to add an exclusion to your anticheat system exactly only to this patch? They have dedicated community discord channel of CVPatch where you can discuss this. 
These are examples how does it look like [Youtube](https://www.youtube.com/channel/UCqR8_RMBZOikS_wF2L8ihWg).

**Comment by Saikur0n** [comment]
Hi. Thank you for your effort to make such a good instruction for community to give a little bit more fresh looking to WoTLK client. I also wanted to make such a post but was waiting when the CVPatch (Collection Visual Patch) by vvladoon would be updated and update was released 31st December. They have related discord channel [CVPatch - Discord](https://discord.com/invite/mmnWAExbXh). Their collection of patches is really good and in every 2-3 month they release new version with new additions and bug fixies. I know russian language and for me it is easier to follow them and their changelog. 
For example they have patch-A.mpq with updated creature models and mounts while Finsternis' Updated Creature Models patch is old and have some bugs.
So, I wanted to ask, should I create a separate post with CVPatch on english in this forum or you will add it to your description? ;) CVPatch original forum post is on russian language and for chromiecraft community it may be difficult to understand it.

**Reply by Marotheit** [reply]
I certainly don't mind, feel free to send me a message on Discord when you wrap up with everything! You'd be my best friend in the world if you provided a changelog!

**Reply by Saikur0n** [reply]
I don't use discrod. I visit and follow CVPatch changes only on their forum (UWOW). I will make here a post translated from original source. By the way, does your patch for login screen works well with CC or it triggers anti cheat system of CC? I know that spells visual patch could trigger it if the spell.dbc in patch modified. I like how you modified login screen with removing unnecessary clutter. And also, is it mandatory to use your modified WoW.exe for your login screen patch?

**Reply by Marotheit** [reply]
I'm not sure if it triggers the anti-cheat, but it's very possible. The executable removes the signature check so you can actually modify the XML layout of the login screen, so it's required to function unfortunately.

**Reply by Marotheit** [reply]
I ended up adding some of those patches in the newest revision of the guide, found [here](https://github.com/Marotheit/A-Guide-to-Modernizing-the-WotLK-Client). The spell effects are pretty stunning!
